lib/ui/painting/image_generator.h

```diff
@@ -20,6 +21,10 @@ namespace flutter {
 /// @see    `ImageGenerator::GetScaledDimensions`
 class ImageGenerator {
  public:
+  /// Frame count value to denote infinite looping.
+  const static unsigned int kInfinitePlayCount =
+      std::numeric_limits<unsigned int>::max();
```

Comment:
The public Dart API for this uses -1 for this. Does this ever propagate back to Dart?

Even if not, it seems like it'd be better to just use -1 for this to be consistent. See https://api.flutter.dev/flutter/dart-ui/Codec/repetitionCount.html

Reply:
I ended up diverting from repetitionCount in [this comment thread](https://github.com/flutter/engine/pull/26338#discussion_r638256354) towards something a little more intuitive. The downside is converting back and forth when interfacing with Skia. (Note that this isn't the only difference -- all the numbers are also shifted by 1: repetitionCount=0 means play once and stop, PlayCount=1 means the same thing).

Reply:
Ahh ok, I guess this makes our tests in the engine repo a little more readable. Ok.